man/docker.1.md

```diff
@@ -425,6 +425,26 @@ device.
 
 Example use: `docker daemon --storage-opt dm.use_deferred_removal=true`
 
+#### dm.use_deferred_deletion
```

Comment:
@moxiegirl 

You have a good point that why this feature is not enabled by default. And only reason I did not do it is that it is very new feature. I will enable it by default in fedora. Once I have confidence that it is working reasonably well (based on fedora usage), I want to propose enabling it by default docker upstream.

So by the way, how do I "mount directory of one container into another container"?